src/main/java/org/openstreetmap/atlas/geography/geojson/parser/README.md

```diff
@@ -231,6 +231,18 @@ This means that for the auto-mapper to work you need to follow the `JavaBean` st
 * 1D arrays of Java Beans.
 * A Map of scalar values in it's `key` and `value`.
 
+#### Parsed Geo JSON Geometry to Atlas Geometry mapping
+
+| Geo Json Geometry        | Atlas Geometry                                                                        |
+| -------------------------|---------------------------------------------------------------------------------------|
+| Point                    | `Location`                                                                            |
+| MultiPoint               | `List<Location>`                                                                      |
+| LineString               | `PolyLine`                                                                            |
+| MultiLineString          | `List<PolyLine>`                                                                      |
+| Polygon                  | `Polygon`                                                                             |
+| MultiPolygon             | `List<Polygon>`                                                                       |
```

Comment:
Any reason this is not going to Atlas `MultiPolygon`?

Reply:
Yes, I had it originally as a MultiPolygon but there is no easy way with Atlas MultiPolygon currently to handle donuts. See comment from Ranek above.